src/AutoRest.CSharp/Common/Input/InputTypes/ConvenienceMethodOmitReason.cs

```diff
@@ -0,0 +1,11 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace AutoRest.CSharp.Common.Input;
+
+internal enum ConvenienceMethodOmitReason
+{
+    SuppressedInTypeSpec = 0,
```

Comment:
Ususally, the enum value of 0 is considered as a place holder or 'None'.  What is the best practise in our codebase?

Reply:
The default value of the corresponding property is null because it is nullable, therefore I did not put a None here.